resources/views/backend/books/list.blade.php

```diff
@@ -76,20 +76,28 @@
                           <th>@sortablelink('borrowings_count', __('books.total_borrow'))</th>
                           <th>{{ __('general.options') }}</th>
                       </tr>
-                    @foreach ($books as $book)
-                          <tr>
-                              <td>{{ $book->id }}</td>
-                              <td>{{ $book->name }}</td>
-                              <td>{{ $book->author }}</td>
-                              <td>{{ $book->avg_rating }}</td>
-                              <td>{{ $book->total_borrow }}</td>
-                              <td align="center">
-                                  <a href="{{ route('books.edit', $book->id) }}"
-                                     class= "btn-edit fa fa-pencil-square-o btn-custom-option pull-left-center"></a>
-                                  <i class="btn btn-danger btn-lg fa fa-trash-o"></i>
-                              </td>
-                          </tr>
-                    @endforeach
+                        @if (count($books) > 0)
+                            @foreach ($books as $book)
+                                  <tr>
+                                      <td>{{ $book->id }}</td>
+                                      <td>{{ $book->name }}</td>
+                                      <td>{{ $book->author }}</td>
+                                      <td>{{ $book->avg_rating }}</td>
+                                      <td>{{ $book->borrowings_count }}</td>
+                                      <td align="center">
+                                          <a href="{{ route('books.edit', $book->id) }}"
+                                             class= "btn-edit fa fa-pencil-square-o btn-custom-option pull-left-center"></a>
+                                          <i class="btn btn-danger btn-lg fa fa-trash-o"></i>
+                                      </td>
+                                  </tr>
+                            @endforeach
+                        @else
+                            <tr>
+                                <td align="center" colspan="6">
+                                    <p class="text-info h1">Sorry, the page you are looking for could not be found.</p>
```

Comment:
trans những text này đi Luân 
Sorry, the page you are looking for could not be found